src/test/resources/hudson/plugins/jobConfigHistory/configuration-as-code.yml

```diff
@@ -0,0 +1,12 @@
+unclassified:
+  jobConfigHistory:
+    excludePattern: "test1\\.xml|test2\\.xml"
+    excludedUsers: "SYSTEM,user1,user2"
+    historyRootDir: "/var/jenkins_home"
+    maxDaysToKeepEntries: "30"
+    maxEntriesPerPage: "20"
+    maxHistoryEntries: "10000"
+    saveModuleConfiguration: false
+    showBuildBadges: "never"
+    showChangeReasonCommentWindow: false
+    skipDuplicateHistory: false
```

Comment:
Git complains about a missing "newline at end of file" here. So you could add one.

Reply:
Done.